### PR TITLE
[5.6] Fix incorrect executable permissions

### LINKED ISSUE #24867: Executable bit set incorrectly on several files
Affected Laravel Versions: current [master (7b2e13047)](https://github.com/laravel/framework/tree/7b2e13047), current [5.6 (23f6279)](https://github.com/laravel/framework/tree/23f6279)
### Description:
A lot of files have the executable bit set, among them several `.md` and `.json` files. Most if not all should not be marked executable.

### Steps To Reproduce:
`git clone https://github.com/laravel/framework.git && find framework -type f -executable`

### PR DESCRIPTION
**Pull request by ght**
Fixes #24867
Several files erroneously had executable permissions.
This commit is the result of running:
```sh
find src tests -type f -executable -exec chmod -x {} +
```

### CODE REVIEW

**Comment by ght** [comment]
@taylorotwell It would be nice if you could comment why this was just closed. If this PR does not follow some guidelines, just tell me what I should change and I will gladly update it.

**Comment by sr2ds** [comment]
I think this problem for enviroment, dont repository files.
That is, it does not need correction, this must be done when the project is installed.

**Comment by tillkruss** [comment]
Try resubmitting this to 5.6. 

Changing permission in a minor release might lead to breaking changes.

**Comment by mfn** [comment]
> Changing permission in a minor release might lead to breaking changes.

This is funny because I know for a fact the permissions within a major release changed multiple times. I'm part of a minority having `vendors/` commited and a I always check the diffs and see every 5 release or so randomly change.

TL;DR: if that might be a breaking change, it already happened multiple times in the past and no one spoke up.

**Comment by ght** [comment]
I resubmitted this against `5.7` as #25191, as suggested by @tillkruss
